events/tests/test_api.py

```diff
@@ -1059,6 +1062,44 @@ def test_cannot_unenrol_from_occurrence_in_past(
     assert_match_error_code(executed, PAST_ENROLMENT_ERROR)
 
 
+@freeze_time("2024-01-01 12:00:00")
+def test_cannot_unenrol_within_48_hours(guardian_api_client, child_with_user_guardian):
+    # Create occurrence 47 hours in future (within 48 hour limit)
```

Comment:
How about setting the explicit value for the environment variable here so the test is robust against local .env file changes?
```suggestion
    settings.KUKKUU_ENROLMENT_UNENROL_HOURS_BEFORE = 48
    # Create occurrence 47 hours in future (within 48 hour limit)
```

Reply:
Without it, it's testing the default behaviour and that's what I think we should use. I have not added the environment variable to pipeline configurations at all, so I'm trusting in default value.